pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -234,6 +234,13 @@
             "#import": [ "rhel", "linux-x64" ]
         },
 
+        "rhel.6": {
```

Comment:
I'd not inherit. Within a major release, APIs and ABIs can be assumed to be stable for a bunch of interesting libraries. Only for core packages, there is a compatibility across a number of major releases.
For RHEL7 you can find some info here https://access.redhat.com/sites/default/files/attachments/rhel7_app_compatibility_wp_1.pdf